test/simulator/main/org/apache/cassandra/simulator/ClusterSimulation.java

```diff
@@ -198,7 +198,7 @@ public static abstract class Builder<S extends Simulation>
         protected HeapPool.Logged.Listener memoryListener;
         protected SimulatedTime.Listener timeListener = (i1, i2) -> {};
         protected LongConsumer onThreadLocalRandomCheck;
-        protected String lwtStrategy = "migration";
+        protected String transactionMode = "migration";
```

Comment:
did you only keep this so paxos running accord doesn't change topologies?  I deleted in my branch but I am working on trying to get topology changes working,

Reply:
I think lwtStrategy doesn't exist anymore? I was probably doing the bare minimum to get it working